config/config.go

```diff
@@ -613,6 +613,7 @@ func SetupViper(v *viper.Viper, filename string) {
 	v.SetDefault("adapters.mgid.endpoint", "https://prebid.mgid.com/prebid/")
 	v.SetDefault("adapters.visx.endpoint", "https://t.visx.net/s2s_bid?wrapperType=s2s_prebid_standard")
 	v.SetDefault("adapters.tappx.endpoint", "https://{{.Host}}")
+	v.SetDefault("adapters.engagebdr.endpoint", "http://dsp.bnmla.com/hb")
```

Comment:
You should also set a default usersync endpoint in the `setDerivedDefaults` function on L419